deployment/frozen/testnet.toml

```diff
@@ -1,5 +1,11 @@
-# version v1
+# version v0.1.0
 [[code_hash_list]]
-commit_hash = "457e32d23b5f401cad837e52a568281784943e18"
 cluster = "598d793defef36e2eeba54a9b45130e4ca92822e1d193671f490950c3b856080"
 spore = "bbad126377d45f90a8ee120da988a2d7332c78ba8fd679aab478a19d6c133494"
+
+# version v0.2.1
+[[code_hash_list]]
+cluster = "7366a61534fa7c7e6225ecc0d828ea3b5366adec2b58206f2ee84995fe030075"
+spore = "5e063b4c0e7abeaa6a428df3b693521a3050934cf3b0ae97a800d1bc31449398"
+cluster_agent = "c986099b41d79ca1b2a56ce5874bcda8175440a17298ea5e2bbc3897736b8c21"
+cluster_proxy = "be8b9ce3d05a32c4bb26fe71cd5fc1407ce91e3a8b9e8719be2ab072cef1454b"
```

Comment:
```bash
# checkout commit 53bfd4d4bbbee773ca10c1181982d9a27718826d tag: 0.2.1
capsule clean
capsule build --release
```

And then I check the code_hashes:

## commit: 53bfd4d (capsule build --release)
- code_hash of contact spore              : 0x7e8bf78a62232caa2f5d47e691e8db1a90d05e93dc6828ad3cb935c01ec6d208
- code_hash of contact cluster            : 0x7366a61534fa7c7e6225ecc0d828ea3b5366adec2b58206f2ee84995fe030075
- code_hash of contact cluster_agent      : 0xc986099b41d79ca1b2a56ce5874bcda8175440a17298ea5e2bbc3897736b8c21
- code_hash of contact cluster_proxy      : 0xbe8b9ce3d05a32c4bb26fe71cd5fc1407ce91e3a8b9e8719be2ab072cef1454b
- code_hash of contact spore_extension_lua: 0x94a9b875911ace20f1f0d063a26495d14e4b04e32fd218261bb747f34e71ae47

### Could you check the code_hash of `contact spore` again?